homeassistant/components/tailwind/cover.py

```diff
@@ -60,6 +60,8 @@ async def async_open_cover(self, **kwargs: Any) -> None:
         The Tailwind operating command will await the confirmation of the
         door being opened before returning.
         """
+        if not self.is_closed:
```

Comment:
This is not correct. This assumes the internal state matches the actual device state, which doesn't have to be true as this is a polling integration. Furthermore, Home Assistant does not have rules or architectural design around these methods to be idempotent.

The issue writes about an uncouth exception; that should be caught instead (although the linked issue is missing the actual stack trace)

Reply:
My thoughts are already described above, confused by your question on my thoughts here